scripts/kokoro-server.py

```
@@ -0,0 +1,56 @@
+
+SOCKET_PATH = "/tmp/kokoro-server.sock"
+
+def load_model():
+    from kokoro import KPipeline
+    print("Loading Kokoro model on GPU...", flush=True)
+    pipeline = KPipeline(lang_code='a')
+    print(f"Model loaded on {next(pipeline.model.parameters()).device}", flush=True)
+    return pipeline
+
+def generate(pipeline, text, output_path):
+    for i, (gs, ps, audio) in enumerate(pipeline(text, voice='af_heart')):
```

Ruff (0.15.0)

[warning] Loop control variable `i` not used within loop body

Rename unused `i` to `_i`

(B007)